### ISSUE TITLE
Accessibility Issue with ColorPicker in High contrast mode

### ISSUE THREAD

**Issue by moizgh** [OfficeDev/office-ui-fabric-react#8393]
![image](https://user-images.githubusercontent.com/25937675/54643862-48d09d00-4a6e-11e9-9b74-f7a7b1dc4e7b.png)

**Comment by khmakoto** [comment]
Hi @moizgh, what high contrast mode are you using? I just checked with high contrast mode enabled on Windows and Edge and, while the picked color area needs to be fixed to show the actual color, the `ColorPicker` shows up correctly as can be seen in the image below:

![image](https://user-images.githubusercontent.com/7798177/54646006-fd60c380-4a5a-11e9-96b0-60865208d534.png)

**Comment by moizgh** [comment]
I am using firefox browser and I switched the high contrast mode from the OS settings.

**Comment by khmakoto** [comment]
@micahgodbolt I think this raises a question of whether the ColorPicker should be High Contrast Accessible in the first place, and if it should, then how would we achieve it?

**Comment by micahgodbolt** [comment]
make sure you're testing high contrast mode in edge only. That is our target browser for high contrast.

**Comment by khmakoto** [comment]
@moizgh Can you check if you see the same thing I do when using Edge?

@micahgodbolt I wonder whether colors should be updated for high contrast in the `ColorPicker` and, if so, how should we approach doing it? Alternatively, we could just have the `ColorPicker` show regular colors regardless of High Contrast or not as it's doing right now, in which case we would only have to fix the color preview on the right of the control. What are your thoughts on this?

**Comment by micahgodbolt** [comment]
the colors should probalby not be affected by high contrast. That setting is to make the UI easier to read. So as long as the UI around the picker adapts to high contrast then we're fine. The actual colors need to be true to the actual color value.

**Comment by khmakoto** [comment]
Perfect, then I'll go ahead and just make the change for the color selection to be visible in High Contrast.

Thanks @micahgodbolt!
